sema4ai/src/sema4ai_code/robocorp_language_server.py

```diff
@@ -1047,6 +1048,42 @@ def m_list_dev_tasks(self, action_package_uri: str) -> partial:
             partial(self._list_dev_tasks_in_thread, action_package_uri)
         )
 
+    def m_get_external_api_url(self) -> str | None:
+        home = get_default_sema4ai_home()
+        external_api_pid = home / "sema4ai-studio" / "external-api-server.pid"
+
+        if not external_api_pid.exists():
+            log.warning(
```

Comment:
I think you should change this to be `info` and not really `warning` (also remove the Warning in the string). This isn't something really that important and it'll appear whenever the user launches something and Studio hasn't been opened (which isn't very nice).

I'd even be ok in removing those loggings.